### PR TITLE
Lad 137 - Javadoc

### PR DESCRIPTION
**Pull request by addemod**

### CODE REVIEW

File: LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/RequestQueueSingleton.kt [hunk 1]
```diff
@@ -21,14 +21,16 @@ class RequestQueueSingleton constructor(context: Context) {
             }
     }
 
-    val requestQueue: RequestQueue by lazy {
+    /**
+     * Return the volley request queue, instantiates and sets a new instance if null
+     */
+    private val requestQueue: RequestQueue by lazy {
```
Comment (Dequilla) [inline LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/RequestQueueSingleton.kt:27]: REEEE NO PARAM

File: LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/RequestQueueSingleton.kt [hunk 2]
```diff
@@ -21,14 +21,16 @@ class RequestQueueSingleton constructor(context: Context) {
...
         Volley.newRequestQueue(context.applicationContext)
     }
 
-    fun <T> addToRequestQueue(req: Request<T>) {
-        requestQueue.add(req)
-    }
-
+    /**
+     * Add a request to the volley request queue
+     */
```
Comment (Dequilla) [inline LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/RequestQueueSingleton.kt]: SHIT NO PARAM

File: LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/CustomRestRequest.kt [hunk 3]
```diff
@@ -13,26 +21,20 @@ class CustomRestRequest (
     errorListener: Response.ErrorListener
 ): StringRequest(method.value, url, listener, errorListener) {
 
+    /**
+     * Returns the response body as a byte array
+     */
```
Comment (Dequilla) [inline LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/CustomRestRequest.kt]: MISSING SNABEL

File: LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/CustomRestRequest.kt [hunk 4]
```diff
@@ -13,26 +21,20 @@ class CustomRestRequest (
...
     override fun getBody(): ByteArray {
         jsonString?.let {
             return it.toByteArray(Charset.forName("utf-8"))
         }
         return ByteArray(0)
     }
 
+    /**
+     * Returns the default body content type
+     */
```
Comment (Dequilla) [inline LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/CustomRestRequest.kt]: MISSING THAT SNABEL

File: LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/SharedPreferencesManager.kt [hunk 5]
```diff
@@ -4,6 +4,9 @@ import android.content.Context
 import android.content.SharedPreferences
 import androidx.core.content.edit
 
+/**
+ * Shared preferences manager to enable saving stuff, such as app settings
+ */
```
Comment (Dequilla) [inline LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/SharedPreferencesManager.kt]: POPERTY MATE

Comment (addemod) [reply LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/util/SharedPreferencesManager.kt]: Excuse me, property*

File: LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/RequestQueueSingleton.kt (hunk 1 above)
Comment (addemod) [reply LaderAppen2/app/src/main/java/se/ju/student/hihe1788/laderappen2/RequestQueueSingleton.kt:27]: PARAM ON VAL??

**Pull request merged**